codeclimate-wrapper

```diff
@@ -28,7 +28,7 @@ invalid_docker_host() {
   invalid_setup "invalid DOCKER_HOST=$host, must be unset or unix:///var/run/docker.sock"
 }
 
-if command -v docker-machine > /dev/null 2>&1; then
+if [ "$DOCKER_MACHINE_NAME" ] && command -v docker-machine > /dev/null 2>&1; then
```

Comment:
I think you meant `[ -n "..." ]` (the test you have is a bashism only supported by `[[`). Also, do we think it's reasonable to go by this check alone and remove the `command` check?